test/PayID/fakes/fake-pay-id-client.ts

```diff
@@ -0,0 +1,21 @@
+import PayIDClientInterface from '../../../src/PayID/pay-id-client-interface'
+import Result from '../../Common/Helpers/result'
+
+/**
+ * A fake Pay ID Client which can return faked values.
+ */
+export default class FakePayIDClient implements PayIDClientInterface {
+  /**
+   * @param xrpAddressResult The object that will be returned or thrown from a call to `xrpAddressForPayID`.
+   */
+  constructor(private readonly xrpAddressResult: result<string | undefined>) {}
```

Comment:
looks like this needs to be `Result<string | undefined>`

Reply:
This is what I get for coding during a sprint planning meeting.